.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,77 @@
+name: Ci suite
+on: [push, pull_request]
+jobs:
+    symfony:
+        name: PHPUnit tests (PHP ${{ matrix.php-versions }})
+        runs-on: ubuntu-latest
+
+        # Docs: https://docs.github.com/en/actions/using-containerized-services
+        services:
+            postgres:
+                image: postgres:latest
+                env:
+                    POSTGRES_USER: postgres
+                    POSTGRES_PASSWORD: postgres
+                    POSTGRES_DB: postgres
+                ports:
+                    - 5432/tcp
+                options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 3
+        strategy:
+            fail-fast: false
+            matrix:
+                php-versions: ['7.4', '8.0']
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+
+            # Docs: https://github.com/shivammathur/setup-php
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-versions }}
+                  tools: phpunit-bridge
+                  extensions: mbstring, xml, ctype, iconv, intl, pdo_sqlite, pgsql
+                  coverage: xdebug
+
+            # Local PostgreSQL service in GitHub hosted environments is disabled by default.
+            # If you are using it instead of service containers, make sure you start it.
+            # - name: Start postgresql service
+            #   run: sudo systemctl start postgresql.service
+
+            - name: Get composer cache directory
+              id: composer-cache
+              run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
+
+            # Without this step `git add .` command will result in exit code 128.
+            # This ensures the comitter info is set, for now prefil with some exiting
+            # variables in GitHub actions.
+
+            - name: Set committer info
+              id: committer-info
+              run: |
+                  git config --global user.email ${GITHUB_ACTOR_ID}+actor_id@noreply.github.com
+                  git config --global user.name $GITHUB_ACTOR
```

Comment:
```suggestion

```

don't see how that's relevant?

Reply:
Well there is a comment on top, describing the issue. Within phpunit and the functional tests we actually create a new git repo and without this we can not run `git add .` (https://github.com/repman-io/repman/blob/c384362816510d267488773077f4dbb73594c035/tests/Unit/Service/Security/SensioLabsSecurityCheckerTest.php#L156C1-L156C1)  it will result in exit code 128 and the next steps just fail silently

Reply:
Oooh, I see, sorry! Thought it was something around workflow, not the actual tests using `git add` :+1: